python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -838,4 +950,6 @@ def process_graph(args):
     argparser.add_argument("--num_partitions", type=int, default=1,
                            help="The number of graph partitions. " + \
                                    "This is only valid if the output format is DistDGL.")
+    argparser.add_argument("--skip_nonexist_edges", action='store_true',
```

Comment:
Which are the cases where this happens? May be better to raise an error, depends on what we expect the user intent to be.

Reply:
some data (e.g., MAG graph) has this problem that some of the end point nodes don't exist in the node files. this option is to tell the pipeline to ignore these edges. by default, it raises an error. however, i would like to give users the option of skipping these edges. otherwise, the users have to deal with this problem themselves.